README.md

```diff
@@ -15,28 +17,52 @@ docker run --rm -t \
            -e VAULT_SECRET_ID=<APPROLE_SECRET_ID> \
            quay.io/app-sre/vault-manager:latest -dry-run
 ```
-Note that running vault-manager with -dry-run flag will only print planned actions,
-remove this flag to make changes enter into effect
+
+Note that running vault-manager with the `-dry-run` flag will only print planned actions;
+remove this flag to make changes to the vault instance.
+
+## Environment Variables
+
+- `VAULT_ADDR`<br>
+URL of the vault instance
+- `VAULT_TOKEN`<br>
+Token to authenticate with the vault instance
+- `VAULT_AUTHTYPE`<br>
+Authentication type to use with the vault instance, currently only `token` and `approle` are supported
+- `VAULT_ROLE_ID`<br>
+Role ID to use with the vault instance, required if `VAULT_AUTHTYPE` is `approle`
+- `VAULT_SECRET_ID`<br>
+Secret ID to use with the vault instance, required if `VAULT_AUTHTYPE` is `approle`
+- `GRAPHQL_SERVER`<br>
+URL of the graphql server
+- `GRAPHQL_QUERY_FILE`<br>
+Path to the graphql query file
+- `GRAPHQL_USERNAME`<br>
+Username to authenticate with the graphql server
+- `GRAPHQL_PASSWORD`<br>
+Password to authenticate with the graphql server
 
 ## Flags
+
 - `-dry-run`, default=false<br>
 runs vault-manager in dry-run mode and only print planned actions
 - `-thread-pool-size`, default=10<br>
 Some operations are running in parallel to achieve the best performance,
 so `-thread-pool-size` determine how many threads can be utilized
 
-# Changing data.json used for testing
+## Changing data.json used for testing
+
 `data.json` within `tests/app-interface` is utilized by the qontract-server created for testing. If schema / query changes are made, this data bundle must be re-generated and committed with the PR. To re-generate: update `SCHEMAS_IMAGE_TAG` within `.env` (make sure to commit this change as well) and execute `make data` within `/tests/app-interface`
 
-# Local Development
+## Local Development
 
-For local development, the script `/local-dev.sh` can be ran to configure necessary resources to mirror testing performed within PR check builds.  
+For local development, the script `local-dev.sh` can be ran to configure necessary resources to mirror testing performed within PR check builds.
 
 Once the script completes, the following containers will be running:
-* keycloak 
+* keycloak
     * necessary for oidc testing
     * view `/tests/keycloak` for configuration files applied to the instance
-* qontract-server 
+* qontract-server
     * view `/tests/app-interface/data/services/vault/config` for all resources being reconciled by tests
 * primary vault instance
     * running on `localhost:8200`
```

Comment:
my senses are tingling about the missing declaration of vault instance number 2 URI + PORT

Reply:
Missing in the README?  It is there:
```
primary vault instance
running on localhost:8200
secondary vault instance
running on localhost:8202
```